pkg/metrics/podlabelmetrics_test.go

```diff
@@ -0,0 +1,71 @@
+package metrics
+
+import (
+	"testing"
+
+	"github.com/opencost/opencost/pkg/clustercache"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestWhitelist(t *testing.T) {
```

Comment:
Any way we could get an assertion here that could fail? Hard to call it a "test" if it can't fail. (Or perhaps I am being dumb and am missing that bit?)

Reply:
ah, this is my bad. good catch, I'll add an assertion.